bundles/org.openhab.binding.ipcamera/README.md

```diff
@@ -0,0 +1,1312 @@
+If the data is coming in faster then it can be processed it will result in an Out Off Memory Error (OOME) that can halt your Openhab server, so if your reading this to shut down the logging, reconsider the need to use the Image channel.
+
+If you still wish to use the Image channel, the following is how to deal with the log output that is created as the raw picture data in text format is ugly and makes the log very hard to read.
+
++ You can switch to only updating the image channel on EVENTS like motion alarms.
++ Turn off (disable) all events from being logged. 
++ Filter out only the events caused by the image changing before they reach the log file.
+
+The openHAB event.log does not allow normal filtering at a binding level due to the log being a pure output from the event bus.
+
+To disable the event.log you can use this command in Karaf console.
+
+
+```
+
+log:set WARN smarthome.event
+
+```
+
+To re-enable just use the same command with INFO instead of WARN.
+
+
+To filter out only the image events, leaving the rest you can do the following:
+
+```
+sudo nano /var/lib/openhab2/etc/org.ops4j.pax.logging.cfg
+```
+
+Inside that file paste the following, save and then reboot.
+
+```
+############ CUSTOM FILTERS START HERE #################
+# event log filter
+log4j2.appender.event.filter.myfilter1.type = RegexFilter
+log4j2.appender.event.filter.myfilter1.regex = .*changed from raw type.*
+log4j2.appender.event.filter.myfilter1.onMatch = DENY
+log4j2.appender.event.filter.myfilter1.onMisMatch = ACCEPT
+################# END OF FILTERS ######################
+
+```
+
+You can specify the item name in the filter to remove just 1 camera, or you can use the above without the item name to remove all events from images updating, which will be for other bindings as well.
+
+
+## Roadmap for further development
+
+Currently the focus is on creating a stable framework that allows all brands to be used in a consistent way, new features that most users wont use are not held as highly as having a stable and consistent binding. 
+After the binding is merged, the extra features can be added over time.
+
+If you need a feature added that is in an API and you can not program, please raise an issue ticket at Github with a sample of what a browser shows when you enter in the URL, it is usually very quick to add features if someone gives a summary of the URL and what their camera returns.
+
+If you wish to contribute then please create an issue ticket first to discuss how things will work before doing any coding. 
+This is for multiple reasons due to needing to keep things CONSISTENT between brands, lower the risk of breaking changes, and also keep the binding easy to maintain.
```

Comment:
I think this can be shorten to one or two sentences, or?

Reply:
Done: Made shorter.